Annex60/Fluid/Movers/BaseClasses/ControlledFlowMachine.mo

```diff
@@ -31,12 +45,11 @@ protected
   Modelica.Blocks.Sources.RealExpression PToMedium_flow(y=Q_flow + WFlo) if  addPowerToMedium
     "Heat and work input into medium"
     annotation (Placement(transformation(extent={{-100,10},{-80,30}})));
-initial equation
-  V_flow_max=m_flow_nominal/rho_default;
+
 equation
   r_V = VMachine_flow/V_flow_max;
-  etaHyd = cha.efficiency(data=hydraulicEfficiency, r_V=r_V, d=hydDer);
-  etaMot = cha.efficiency(data=motorEfficiency,     r_V=r_V, d=motDer);
+  etaHyd = cha.efficiency(per=per.hydraulicEfficiency, V_flow=VMachine_flow, d=hydDer, r_N=1, delta=1E-4);
+  etaMot = cha.efficiency(per=per.motorEfficiency,     V_flow=VMachine_flow, d=motDer, r_N=1, delta=1E-4);
   dpMachine = -dp;
   VMachine_flow = -port_b.m_flow/rho_in;
```

Comment:
Why is `VMachine_flow = - port_b.m_flow/rho_in` and not `=port_a.m_flow/rho_in`? `rho_in` is the density of inlet fluid...